contracts/utils/TokenManagerDeployer.sol

```diff
@@ -33,5 +34,8 @@ contract TokenManagerDeployer is ITokenManagerDeployer, Create3 {
         tokenManager = _create3(bytecode, tokenId);
 
         if (tokenManager.code.length == 0) revert TokenManagerDeploymentFailed();
+
+        if (implementationType == uint256(TokenManagerType.LOCK_UNLOCK) || implementationType == uint256(TokenManagerType.LOCK_UNLOCK_FEE))
```

Comment:
Can we do this in the service instead? Decouples approval setup (which is meant to be called by the service only and not TokenManagerDeployer) and removes dependency on ITokenManager.

Reply:
done